sample-application/sample-application/app/src/main/java/com/example/samplecredentialwallet/ui/home/HomeScreen.kt

```
@@ -0,0 +1,362 @@
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(12.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            val faceImage = parsedData["faceImage"]
+            if (faceImage != null) {
+                val bitmap = remember(faceImage) {
+                    try {
+                        val imageBytes = Base64.decode(faceImage, Base64.DEFAULT)
+                        BitmapFactory.decodeByteArray(imageBytes, 0, imageBytes.size)
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)